main.go

```diff
@@ -103,6 +103,35 @@ var defaultWatchRepositoriesSync = []string{
 	"mender-qa",
 }
 
+// List of repositories which runs integration tests, but do not need a fresh
+// built client
+var prebuiltClientRepositories = []string{
```

Comment:
I think having a new list of repos is not wise: adding any repository we need to modify both `defaultWatchRepositoriesPipeline` and this one.